index.rst

```diff
@@ -2,8 +2,8 @@
 JWST Data Analysis Tool Notebooks
 =======================================================
 
-The `jdat_notebooks <https://github.com/spacetelescope/jdat_notebooks>`_
-repository contains notebooks illustrating workflows for post-pipeline
+The `jdat_notebooks repository <https://github.com/spacetelescope/jdat_notebooks>`_
+ contains notebooks illustrating workflows for post-pipeline
```

Comment:
When I view the file, I'm seeing a line break after 'jdat_notebooks_repository', and another after "post-pipeline"

Reply:
Thank you! It was a formatting issue in my file. Fixed